src/web/src/helpers.cc

```diff
@@ -419,6 +419,19 @@ check_user_permissions (
 }
 
 char*
+get_current_db_initialized_file (void)
+{
+    char *current_db_initalized_file = NULL;
+    char *env = getenv (EV_DB_INITIALIZED_DIR);
+
+    int rv = asprintf (&current_db_initalized_file, "%s/fty-db-ready", env ? env : "/var/run/");
+    if ( rv == -1 ) {
+        return NULL;
```

Comment:
I wonder if we should also check `if (current_db_initalized_file) free(current_db_initalized_file )` here just in case?